### ISSUE TITLE
Negative Probabilities for sentiment classification

### ISSUE THREAD

**Issue by grecosalvatore** [google-research/bert#765]
I am trying to do sentiment classification on IMDB dataset using the [official notebook](https://colab.research.google.com/github/google-research/bert/blob/master/predicting_movie_reviews_with_bert_on_tf_hub.ipynb#scrollTo=j0a4mTk9o1Qg) .
Why probabilities for both labels are negative? The two probabilities don't need to be [0,1] and the sum =1 ?
the last layer added is a softmax.
thanks

**Comment by elzinga** [comment]
Those negative values appear to correspond to the inputs to a softmax function rather than to the outputs.

**Comment by grecosalvatore** [comment]
I have solved this issue by adding the following line:
probs = tf.nn.softmax(logits, axis=-1)

and i have returned in the model the Probabilities instead of Log_Probabilities
